types/react-data-grid/index.d.ts

```diff
@@ -62,6 +62,14 @@ declare namespace AdazzleReactDataGrid {
          * @param e Information about the event
          */
         onCellsDragged?: (e: CellDragEvent) => void
+        /**
+         * Invoked when the user clicks on one cell to expand it.
+         * @param e Information about the event
+         */
+        onCellExpand?: (e:CellExpandEvent) => void
```

Comment:
There's a more specific type we can use here
```suggestion
        onCellExpand?: (e:CellExpandEvent<T>) => void
```